target/linux/mvebu/image/cortex-a53.mk

```diff
@@ -2,42 +2,51 @@ ifeq ($(SUBTARGET),cortexa53)
 
 define Device/globalscale_espressobin
   $(call Device/Default-arm64)
-  DEVICE_TITLE := ESPRESSObin (Marvell Armada 3700 Community Board)
+  DEVICE_VENDOR := Marvell
+  DEVICE_MODEL := ESPRESSObin (Armada 3700 Community Board)
```

Comment:
This again raises the question whether the brackets should become an alternate name.

Reply:
No idea. Feel free to suggest something.

Reply:
I think you should add
DEVICE_ALT0_VENDOR := Marvell
DEVICE_ALT0_MODEL := Armada 3700 Community Board

Despite, there are several variants below. So, we need one for this device.
Since I have no better idea, I would call it "Non-eMMC"